style.css

```diff
@@ -492,24 +505,64 @@
     width: 300px;
   }
 
+  #nav-bar-container {
+    width: 100%;
+  }
+
   #side-navbar-container li {
     font-family: 'Poppins', sans-serif;
     font-style: normal;
     font-weight: 500;
     font-size: 15px;
     line-height: 20px;
     color: #344563;
+    cursor: pointer;
+  }
+
+  #side-navbar-container li:hover {
+    color: #6070ff;
+  }
+
+  #side-navbar-container li:active {
+    color: #2230d2;
+  }
+
+  .view-project-button button {
+    cursor: pointer;
+  }
+
+  .view-project-button button:hover {
+    color: #fff;
+    background-color: #6070ff;
+    border: none;
+  }
+
+  .view-project-button button:active {
+    color: #fff;
+    background-color: #2230d2;
+    border: none;
+  }
+
+  main {
+    height: auto;
+    border: 1px solid #dfe1e6;
+    border-radius: 0 0 0 100px;
+    background-image: url(./Images/Header\ bg\ 0.75.png);
```

Comment:
- [ ] Kindly add the background colour as suggested by your previous code reviewer, as I can see that you have not made the changes yet. Please adhere to the `Figma` design and try to make your portfolio looks similar to `the template design because having a good colour combination makes your design more consistent, professional and appealing.
- [ ] Please I would like to suggest you in order to complete your project please use the *background-image* on the `.main-container` class and use the `background-color` on this `main` tag. Kindly note that once you add the background-color to the main tag, your project cards look weird but don't worry you just have to add the *background-color: white;* on your project card and the rest of your project will look exactly like the `Figma` design template.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-03 11-30-09](https://user-images.githubusercontent.com/66476329/235843356-844ffbc9-64b5-4251-8ecd-bf1cfdd9f4d0.png)

</td>
<td>

![Screenshot from 2023-05-03 11-21-24](https://user-images.githubusercontent.com/66476329/235843384-6b577f5a-ef39-4295-abec-634479f56684.png)

</td>
</tr>
</table>

Reply:
@skyv26 With all due respect, Do I need to remove background color of the project card  image

Reply:
I put the background color on the work section which wraps all the project sections and I add all of the project section cards to color #FFFFFF. It seems fine . Can you please give me a look . 

![Screenshot (22)](https://user-images.githubusercontent.com/102011195/235858169-aa47d8d6-7fbc-4e5c-83a3-f890c056dc20.png)

Reply:
![Screenshot (20)](https://user-images.githubusercontent.com/102011195/235858322-d6d38d59-a7d5-4c45-9f64-3ccffa0210f6.png)

Reply:
@HtetWaiYan7191 Great work I can see you implemented the things. but yes please remove the background from the project card image.

Reply:
Thank you for your clear instructions. 😄